test/libethereum/BlockChainTests.cpp

```diff
@@ -34,10 +34,13 @@ using namespace json_spirit;
 using namespace dev;
 using namespace dev::eth;
 
+
 namespace dev {
 
 namespace test {
 
+eth::Network TestBlockChain::sealEngineNetwork = eth::Network::FrontierTest;
```

Comment:
I change the network by reading blocknumber in test fillers.
this cmd option  (seal engine) still needed for user defined tests
this paricular line is just defult value for static variable for blockchain tests.